manubot/process/tests/test_ci.py

```diff
@@ -9,50 +9,92 @@
 )
 
 
-@pytest.mark.skipif('TRAVIS' not in os.environ, reason='tests environment variables set by Travis builds only')
-def test_get_continuous_integration_parameters_travis():
-    info = get_continuous_integration_parameters()
-    assert info is not None
-    assert info['provider'] == 'travis'
-    assert info['repo_slug'] == 'manubot/manubot'
-    assert info['repo_owner'] == 'manubot'
-    assert info['repo_name'] == 'manubot'
-    assert info['commit']
-    assert info['triggering_commit']
-    assert info['build_url'].startswith('https://travis-ci.com/manubot/manubot/builds/')
-    assert info['job_url'].startswith('https://travis-ci.com/manubot/manubot/jobs/')
-    # test add_manuscript_urls_to_ci_params
-    info_updated = add_manuscript_urls_to_ci_params(info)
-    assert info is info_updated
-    assert re.fullmatch(
-        pattern=r"https://manubot\.github\.io/manubot/v/[0-9a-f]{40}/",
-        string=info['manuscript_url'],
-    )
+@pytest.fixture
+def info():
+    return get_continuous_integration_parameters()
```

Comment:
Is the fixture rerun for every class method where its invoked or is the same info output reused? This is a general question about how fixtures work.

Reply:
It should be a rerun, I think, but it is not expensive - should not be slowing the tests too much.